src/js/jsx/sections/export/ExportPanel.jsx

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2014 Adobe Systems Incorporated. All rights reserved.
+ *  
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"), 
+ * to deal in the Software without restriction, including without limitation 
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense, 
+ * and/or sell copies of the Software, and to permit persons to whom the 
+ * Software is furnished to do so, subject to the following conditions:
+ *  
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *  
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, 
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER 
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING 
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER 
+ * DEALINGS IN THE SOFTWARE.
+ * 
+ */
+
+
+define(function (require, exports, module) {
+    "use strict";
+
+    var React = require("react"),
+        classnames = require("classnames");
+
+    var os = require("adapter/os");
+
+    var TitleHeader = require("jsx!js/jsx/shared/TitleHeader"),
+        LayerExports = require("jsx!js/jsx/sections/export/LayerExports"),
+        strings = require("i18n!nls/strings"),
+        synchronization = require("js/util/synchronization");
+
+    var ExportPanel = React.createClass({
+        /**
+         * A throttled version of os.setTooltip
+         *
+         * @type {?function}
+         */
+        _setTooltipThrottled: null,
+
+        componentWillMount: function () {
+            this._setTooltipThrottled = synchronization.throttle(os.setTooltip, os, 500);
+        },
+
+        /**
+         * Selects the content of the input on focus.
+         * 
+         * @private
+         * @param {SyntheticEvent} event
+         */
+        _handleFocus: function (event) {
+            event.target.scrollIntoViewIfNeeded();
+            if (this.props.onFocus) {
+                this.props.onFocus(event);
+            }
+        },
+
+        shouldComponentUpdate: function (nextProps) {
+            if (this.props.disabled !== nextProps.disabled) {
+                return true;
+            }
+
+            if (!nextProps.visible && !this.props.visible) {
+                return false;
+            }
+
+            return true;
+        },
+
+        /**
+         * Workaround a CEF bug by clearing any active tooltips when scrolling.
+         * More details here: https://github.com/adobe-photoshop/spaces-design/issues/444
+         *
+         * @private
+         */
+        _handleScroll: function () {
+            this._setTooltipThrottled("");
```

Comment:
That thought occurred to me as I was blindly copying this stuff from another Panel component.  What do you think about me creating an issue, and tagging eric in it for his input?